### ISSUE TITLE
Ubuntu is the only DEB system supported :(

### ISSUE THREAD

**Issue by H4kunaM4tata** [nkallen/plasticity#169]
As the tittle says, I tried plasticity as soon as it was released hoping to replace both FreeCAD and Blender and well, things didn't went as expected.
I contacted the support via e-mail just to get deeply disappointed.
The e-mail like shown below feel like "that is not our business". Short and total lack of care.

![image](https://user-images.githubusercontent.com/35049512/236986294-34e0be66-8121-4181-9527-181f1f0a3bbf.png)

Anyway, I am hoping somebody here can point me into the right direction.
When I try to run it via ordinary user, that is all I get.
Google says that could be the software trying to access the memory it has no access to:

![image](https://user-images.githubusercontent.com/35049512/236986162-a253e01a-2d65-4401-9f95-a4f4359b2053.png)

But...... if I run it as root: `plasticity --no-sandbox`
I do have a better output but I have no much idea of what is happening to try to solve the problem myself, if I can.

![image](https://user-images.githubusercontent.com/35049512/236986893-320f9be1-c06b-41f2-9335-e9950397be9d.png)

OS: Mint Cinnamon 20.3
Kernel: 5.15.x
Processor: AMD Ryzen 7 5800H 
Memory: 32GB
GPU: RTX 3050 TI Laptop

Thank you all

**Comment by H4kunaM4tata** [comment]
If that adds any weight to the problem, I wanna try it first prior to buying the license.
I have seen what it can do, it is awesome but so far I have had nothing but bad luck with both the software and the company.

**Comment by Fairlight8** [comment]
First of all, please know that the project is run by a single developer, not a company. I guess he has priorities, and can't support all possible systems.

That said, it probably looks like a problem of dependencies, possible something GPU-driver related.

**Comment by H4kunaM4tata** [comment]
> First of all, please know that the project is run by a single developer, not a company. I guess he has priorities, and can't support all possible systems.
> 
> That said, it probably looks like a problem of dependencies, possible something GPU-driver related.

Thank you for the single developer heads-up. I thought it was a company.

I've had no GPU issues issues before, Blender for what matter works fine.
I'll see if I can find the Linux gang group helping with testing to get this puppy supported with more system.

Thank you

**Comment by Fairlight8** [comment]
No problem. Unfortunately, I don't use deb systems and I don't know what kind of dependencies are missing there, I am sorry. But I do think it's more than possible to run Plasticity in mint, of course. Let me know if I can help you with that.

**Comment by H4kunaM4tata** [comment]
For what matters, Mint 20.3 is equivalent to Ubuntu 20.4 while Ubuntu 22 which is supported by Plasticity, is equivalent to Mint 21.1

While helping the guys to have a Linux version of the slicer for my 3D printer, "outdated" system experienced some odd issues. 

Basically, by compiling it with a latest Ubuntu version, "old" system won't be compatible forcing a whole system upgrade, so they had to use Ubuntu 20.4 instead to generate the DEB package solving the inconsistencies.

I'll do some tests tomorrow with a Mint VM and report back in case others find this.

Thank you.

**Comment by H4kunaM4tata** [comment]
@Fairlight8 like I mentioned before, because it was compiled with a "new" DEB system, older DEB system won't be able to run it.
I fresh installed Mint to 21.1 which is equivalent to Ubuntu 22, it now works just fine.

But I got another problem, I tried it yesterday on a Windows PC and I cannot use on this PC with "Your free trial has expired"
I believe that there is a hardware ID check :(
Will try to contact him. Oh boy.

**Comment by Fairlight8** [comment]
I see, it makes sense. Why can't you update to 21.1? I didn't understand why you needed to maintain the older version. Something about a slicer for a 3D printer?

WHat version did you install on your WIndows machine?

**Comment by H4kunaM4tata** [comment]
I've Linux Mint 21.1 installed now, I did a fresh install this morning just to be able to use Plasticity, got it working but that isn't the point imo. Softwares have to be back compatible, imagine if in order to run it on Windows, users will have to upgrade to Windows 11.

But like you said, single developer taking care of this badass tool so all good.

Mint 20.3 was running just fine, I usually keep a version behind. That's why the old system still.

I tried on Windows 10, my anxiety to try some 3d printing modeling couldn't wait to sort the issue with Linux hahaha

I just received an email from them, hopefully I can sort the trial expired thing and start using it.

Thank you
